.github/workflows/tests.yaml

```diff
@@ -0,0 +1,46 @@
+on: [pull_request]
```

Comment:
Everything looks ok, but still not running 😕 

It might be that a first workflow must exist on master before any actions on PRs work. In which case it might be an idea to add the `workflow_dispatch` event so we can manually trigger after this is merged, in order to check it works.
```suggestion
on: [pull_request, workflow_dispatch]
```

Reply:
> It might be that a first workflow must exist on master before any actions on PRs work.

I was under the impression that this _was_ the case.

Reply:
(Unresolving as this might be useful info to keep visible)